clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -228,46 +229,23 @@ func (p *probesMeasurement) checkProbesReady() (bool, error) {
 		p.framework.GetClientSets().GetClient(), isProbeTarget, expectedTargets)
 }
 
-func isProbeTarget(t prometheus.Target) bool {
-	return t.Labels["namespace"] == probesNamespace
-}
-
-func parseThresholds(params map[string]interface{}) (map[string]*measurementutil.LatencyMetric, error) {
-	thresholds := make(map[string]*measurementutil.LatencyMetric)
-	t, ok := params["thresholds"]
-	if !ok {
-		return thresholds, nil
-	}
-	for name, thresholdVal := range t.(map[string]interface{}) {
-		threshold, err := time.ParseDuration(thresholdVal.(string))
-		if err != nil {
-			return nil, err
-		}
-		thresholds[name] = makeLatencyThreshold(threshold)
+func (p *probesMeasurement) createSummary(latency measurementutil.LatencyMetric) (measurement.Summary, error) {
+	content, err := util.PrettyPrintJSON(&measurementutil.PerfData{
+		Version: p.config.MetricVersion,
+		// NOTE(oxddr): for backward compatibility we need to pass "Probes" to be used a metric label.
```

Comment:
Backward compatibility of data we store in gcs?
Is it so important?

I actually think that it's fine to break it (given we're not relying on it anywhere) to make it readable for other probes added in the future.

Reply:
Yes, more precisely with perf-dash configuration. It just makes my life easier, since I'm going to look at this data. But maybe it's not worth to sacrifice long-term benefit for a short-term comfort.